### PR TITLE
WIP: add rules explorer

### PR DESCRIPTION
**Pull request by Issif**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area rules

> /area registry

> /area build

> /area documentation

**What this PR does / why we need it**:

This PR adds https://github.com/Issif/falco-rules-explorer with the CI to this repo. After the merge, the UI will be accessible as GH Pages website. The index.json is stored on S3 aside the .tar.gz rules artifacts.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/explorer_index.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: "arn:aws:iam::292999226676:role/terraform-20230120142903096000000002"
+          aws-region: ${{ env.AWS_S3_REGION }}
+
+      - name: S3 Sync
+        uses: ItsKarma/aws-cli@v1.70.0
+        with:
+          args: s3 sync --delete --acl public-read rules_explorer/index.json s3://${{ env.AWS_S3_BUCKET }}/${{ env.AWS_S3_PREFIX }}/index.json
```
Comment (FedeDP) [inline .github/workflows/explorer_index.yaml:40]: Who is uploading the index.html file?

Comment (Issif) [reply .github/workflows/explorer_index.yaml:40]: It stays in Github, and we set Github Pages to display /rules_explorer. Only the index.json is stored in S3.

Comment (FedeDP) [reply .github/workflows/explorer_index.yaml:40]: Oh makes sense now, thanks!

**Review by darryk10 — APPROVED** [review]
LGTM

**Comment by leogr** [comment]
cc @LucaGuerra

**Comment by Issif** [comment]
Following the KubeCon, I've some changes to do. I'm also thinking I'll add the frontend in falco-website directly and only generate the json in this repo. Let's discuss about what we want to do.

**Review by leogr — COMMENTED** [review]
> I'm also thinking I'll add the frontend in falco-website directly and only generate the json in this repo. 
@Issif 
It seems like a great idea to me! :+1:

**Comment by Andreagit97** [comment]
any updates here?

**Comment by Issif** [comment]
@Andreagit97 We agreed with @leogr to wait for the new rule framework to manage the maturity. It will update this PR once everything is set.

**Pull request closed without merging**